handler/rest.go

```diff
@@ -132,4 +132,37 @@ func (h ArticleHandler) Delete(c echo.Context) (err error) {
 	}
 
 	return c.JSON(http.StatusCreated, deletedID)
+}
+
+func (h ArticleHandler) Update(c echo.Context) (err error){
+	articleID := c.Param("id")
+	var item models.Article
+	err = c.Bind(&item)
+	if err != nil {
+		resp := ErroResponse{
+			Message: err.Error(),
+		}
+		return c.JSON(http.StatusUnprocessableEntity, resp)
+	}
+
+	query := `UPDATE article SET title=?, body=? WHERE id=?`
+	
+	dbRes, err := h.DB.Exec(query, item.Title, item.Body, articleID)
+	if err != nil {
+		resp := ErroResponse{
+			Message: err.Error(),
+		}
+		return c.JSON(http.StatusInternalServerError, resp)
+	}
+
+	updatedID, err := dbRes.RowsAffected()
```

Comment:
This function won't return `updatedID`, but a total of items that updated.
So for update case, you only check for total updated items. 

So in line 166 you maybe you should change into like this.

```go
if totalUpdated <=0 {
  resp := ErroResponse{
	      Message: "Nothing is changed",
	    }
  return c.JSON(http.StatusInternalServerError, resp)
}

```

Reply:
Yeah I was confused about the output in postman before I update my code. The output:
```json
{
    "id": "1",
    "title": "UpdateKuy",
    "body": "Kuylah Update"
}
```
The action is `PUT: localhost:9090/articles/5` -> `ID` is `5`

After I update my code with above, the output show this:
```json
{
    "id": "",
    "title": "Kuy",
    "body": "Kuylah"
}
```
Not showing the updated article's `ID`

Reply:
Should be commented in the next review